tests/queries/0_stateless/02423_ddl_for_opentelemetry.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env bash
+# Tags: zookeeper
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# The test cases in this file cover DDLs running on both Replicated database engine and non-Replicated database engine.
+# Since the processing flow is a little bit different from each other, in order to share same reference file,
+# we compare the expected result and actual result by ourselves. See check_span method below for more detail.
+
+# This function takes following arguments:
+# $1 - OpenTelemetry Trace Id
+# $2 - Query
+# $3 - Query Settings
+function execute_query()
+{
+    # Some queries are supposed to fail, use -f to suppress error messages
+    echo $2 | ${CLICKHOUSE_CURL_COMMAND} -q -s --max-time 180 \
+                -X POST \
+                -H "traceparent: 00-$1-5150000000000515-01" \
+                -H "tracestate: a\nb cd" \
```

Comment:
FWIW this is not a test for new line, for this you need to use `$'foo\nbar` syntax, but you cannot pass new line in headers, I've converted the test to `clickhouse-client` in fix for escaping - #50045